dex/testing/loadbot/loadbot.go

```diff
@@ -698,3 +715,28 @@ func loadNodeConfig(symbol, node string) map[string]string {
 	}
 	return cfg
 }
+
+func symmetricWalletConfig(numCoins int, midGap uint64) (
+	minBaseQty, maxBaseQty, minQuoteQty, maxQuoteQty uint64) {
+
+	minBaseQty = uint64(maxOrderLots) * uint64(numCoins) * lotSize
+	minQuoteQty = calc.BaseToQuote(midGap, minBaseQty)
+	// Ensure enough for registration fees.
+	if minBaseQty < 2e8 {
+		minBaseQty = 2e8
+	}
+	if minQuoteQty < 2e8 {
+		minQuoteQty = 2e8
+	}
+	// eth fee estimation calls for more reserves.
+	if quoteSymbol == eth {
+		add := (ethRedeemFee + ethInitFee) * uint64(maxOrderLots)
+		minQuoteQty += add
+	}
+	if baseSymbol == eth {
```

Comment:
I guess we need a `// TODO: polygon` here?

Reply:
Added... Is polygon an eth token? I guess our current eth token is also not handled correctly here as it need more eth funds for the redeems and refunds.

Reply:
> Added... Is polygon an eth token?

For practical purposes it's an independent coin.  Behind the scenes it's anchored to ETH, but that is irrelevant for DEX.  It's not a token.

> I guess our current eth token is also not handled correctly here as it needs more eth funds for the redeems and refunds.

Guess so.  Not sure how to resolve or if it's critical to do so.

Reply:
String matching with a possible TODO to add another error type to server.